tests/integration/cqlengine/query/test_queryset.py

```diff
@@ -516,6 +530,15 @@ def test_indexed_field_can_be_queried(self):
         q = IndexedTestModel.objects(test_result=25)
         assert q.count() == 4
 
+        q = IndexedTestModel.objects.filter(test_list__contains=42)
```

Comment:
I know the existing codebase uses bare asserts. We didn't revise all tests when we integrated, but I would prefer not to propagate, and usually try to [fix as I touch a given test](https://github.com/datastax/python-driver/commit/a09c7c41ec9df725234279ed9b0350380b3915c2).